spec/features/facets_spec.rb

```diff
@@ -20,4 +20,16 @@
       expect(page).to have_css("#facet-date_range_sim .missing", text: /Unknown/, visible: false)
     end
   end
+
```

Comment:
@eliotjordan does this test the new order? We probably want to check the text of `h3.facet-field-heading:first-child button` is "Access", right?

Reply:
Yes. Good catch.